src/inference/src/os/cpu_map_info.hpp

```diff
@@ -24,16 +24,24 @@ class CPU {
     int _numa_nodes = 0;
     int _sockets = 0;
     int _cores = 0;
+    std::vector<std::vector<int>> _org_proc_type_table;
     std::vector<std::vector<int>> _proc_type_table;
     std::vector<std::vector<int>> _cpu_mapping_table;
     std::mutex _cpu_mutex;
-    int _plugin_status = PLUGIN_USED_START;
+    std::mutex _plugin_mutex;
```

Comment:
Why two different mutexes are required?

Reply:
_plugin_mutex is used in `reserve_available_cpus` which is provided to other plugin calls. So it guarantees the synchronization of CPU reservation.
_cpu_mutex is used to guarantees synchronization problem of reading and writing of _proc_type_table.

Reply:
The question is still there: why cannot we use single mutex to hadle all critical sections? Do we really need to put all `reserve_available_cpus` logic under mutex? As I can see it modifies `cpu._proc_type_table` which is one critical sections, but what else?

Reply:
Ok, I removed _plugin_mutex.